Services/MailgunTransport.php

```diff
@@ -103,7 +103,14 @@ public function send(Swift_Mime_Message $message, &$failedRecipients = null)
 
         $postData = $this->getPostData($message);
         $domain = $this->getDomain($message);
-        $result = $this->mailgun->sendMessage($domain, $postData, $message->toString());
+        try {
+            $result = $this->mailgun->sendMessage($domain, $postData, $message->toString());
+        } catch (\Exception $e) {
+            $failedRecipients[] = $message->getHeaders()->get('To');
```

Comment:
All recipients are in the $postData['to']. See https://github.com/tehplague/swiftmailer-mailgun-bundle/blob/0.2.1/Services/MailgunTransport.php#L165